diag_manager/fms_diag_axis_object.F90

```diff
@@ -384,6 +484,38 @@ subroutine check_if_valid_edges(edges)
        call mpp_error(FATAL, "diag_axit_init: The edge axis has not been defined. "&
                              "Call diag_axis_init for the edge axis first")
   end subroutine check_if_valid_edges
+
+  !> @brief Loop through a variable's axis_id to determine and return the domain type and domain to use
```

Comment:
I still think this description is lacking.  The \brief describes what this subroutine does, but there is no explanation for it's use.  I think with the next update, you should have a \description instead of \brief that explains why you use this.  In the larger plan, this routine is necessary.  If you're just looking at this module, you will scratch your head and wonder why this exists.